src/server/exchangeToken.ts

```diff
@@ -0,0 +1,68 @@
+import Axios from 'axios'
+import Router from 'koa-router'
+import { ParameterizedContext } from 'koa'
+import { createSession } from 'shared/sessionStorage'
+import { ServerCookieStorage } from 'utils/storage/ServerCookieStorage'
+import { WithLoggerState } from './middleware/enhancers'
+
+enum SearchParams {
+  AuthorizationCode = 'authorizationCode',
```

Comment:
<i class='graphite__hidden'>[Re: lines 9 to 9]<br/></i>

Didn't you define this as `authorization_code` in racoon?<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/HedvigInsurance/web-onboarding/1210?utm_source=unchanged-line-comment">Graphite</a>.</span>

Reply:
Good catch!